spec/models/period_contribution_calculator_spec.rb

```diff
@@ -108,5 +108,43 @@
         expect(period_contribution.tax_relief).to eq(1.72)
       end
     end
+
+    context 'when employee is contributing more than 40,000 per year' do
+      let(:eligible_salary) { 80_000 }
+      let(:employee_percent) { 60 }
+      let(:employer_percent) { 1 }
+
+      context 'when yearly frequency' do
+        let(:salary_frequency) { 1 }
+
+        it 'returns employee tax relief' do
+          expect(period_contribution.tax_relief).to eq(8000)
+        end
+      end
+
+      context 'when monthly frequency' do
```

Comment:
All these tests looks similar. Is it possible to trim it down? You could think about using _shared examples_. You may also opt to iteratively consider all possibilities with one block of code fed with a hash/array of all input data.